config_create.go

```diff
@@ -37,7 +41,21 @@ func init() {
 	configCmd.AddCommand(configCreateCmd)
 }
 
+// configExists checks whether a configuration file has already been created.
+// Returns false if not true if it already exists.
+func configExists(path string) bool {
+	info, err := os.Stat(path)
+	if os.IsNotExist(err) || os.IsPermission(err) {
+		return false
+	}
+	return !info.IsDir()
+}
+
 func configCreate(cmd *cobra.Command, args []string) error {
 	cluster := cluster.New(defaultConfig)
+	if configExists(configCreateOptions.file) {
+		message := fmt.Sprintf("Configuration file at %s already exists...", configCreateOptions.file)
```

Comment:
There's a `fmt.Errorf` to combine those two lines in one

Reply:
For some reason I always forget Errorf. :(